package-lock.json

```diff
@@ -1,21 +1,27 @@
 {
   "name": "kbar",
-  "version": "0.1.6",
+  "version": "0.1.0-beta.1",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "kbar",
-      "version": "0.1.6",
+      "version": "0.1.0-beta.1",
       "license": "MIT",
       "devDependencies": {
+        "@reach/portal": "^0.16.0",
         "@types/react": "^17.0.19",
         "@types/react-dom": "^17.0.9",
         "@types/react-router-dom": "^5.1.8",
         "css-loader": "^6.2.0",
         "esbuild-loader": "^2.15.1",
+        "fast-equals": "^2.0.3",
         "font-loader": "^0.1.2",
+        "match-sorter": "^6.3.0",
         "prism-react-renderer": "^1.2.1",
+        "react": "^16.0.0 || ^17.0.0",
+        "react-dom": "^16.0.0 || ^17.0.0",
+        "react-router-dom": "^5.3.0",
```

Comment:
Hope we can use dependency injection for this as a future improve, lots of folks not using this specific version